src/main/scala/no/ndla/conceptapi/controller/ConceptController.scala

```diff
@@ -61,48 +62,42 @@ trait ConceptController {
         case Some(scroll) =>
           conceptSearchService.scroll(scroll, language) match {
             case Success(scrollResult) =>
-              val responseHeader = scrollResult.scrollId.map(i => this.scrollId.paramName -> i).toMap
-              Ok(searchConverterService.asApiConceptSearchResult(scrollResult), headers = responseHeader)
+              Ok(searchConverterService.asApiConceptSearchResult(scrollResult), getResponseScrollHeader(scrollResult))
             case Failure(ex) => errorHandler(ex)
           }
         case None => orFunction
       }
 
+    private def getResponseScrollHeader(result: SearchResult[_]) =
+      result.scrollId.map(i => this.scrollId.paramName -> i).toMap
+
     private def search(query: Option[String],
                        sort: Option[Sort.Value],
                        language: String,
                        page: Int,
                        pageSize: Int,
                        idList: List[Long],
-                       fallback: Boolean) = {
+                       fallback: Boolean,
+                       subjectIds: Set[String]) = {
```

Comment:
Treng å kunne søke på tags som parameter. For eksempel gi meg alle forklaringer med tag "Verktøy". Det er ikkje så aktuellt å hente ut alle forklaringer for Norsk og Naturfag, men greit å ha muligheten.

Reply:
Will fix